src/page/generate/ViewDocumentPage.tsx

```
@@ -1148,17 +1002,17 @@ Silakan lanjutkan pembuatan bagian berikutnya dari dokumen tanpa mengulangi kont
           const signatories = documentData?.metadata?.parties && documentData.metadata.parties.length > 0
-                             ? documentData.metadata.parties
-                             : [
-                                 { name: "Pihak Pertama", position: "(Nama & Jabatan)" },
-                                 { name: "Pihak Kedua", position: "(Nama & Jabatan)" },
-                               ];
-          
+            ? documentData.metadata.parties
+            : [
+              { name: "Pihak Pertama", position: "(Nama & Jabatan)" },
+              { name: "Pihak Kedua", position: "(Nama & Jabatan)" },
+            ];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)